main.go

```diff
@@ -98,6 +99,7 @@ func main() {
 			for {
 				prs, resp, _ := client.Search.Issues(
 					// Search query to find PR's
+					context.Background(),
```

Comment:
If you prefer, it's completely okay to keep your code as is, just use `context.Background()` for now. It's valid and will continue to behave as before.

Later on, once you have more advanced needs and you want to take advantage of things that contexts allow you to do, then you can change this code to propagate another context.